compiler/circle-mpqsolver/src/CircleMPQSolver.cpp

```diff
@@ -129,11 +127,12 @@ int entry(int argc, char **argv)
     return EXIT_FAILURE;
   }
 
-  VERBOSE(l, 0) << ">> Searching mixed precision configuration " << std::endl
-                << "model:" << input_model_path << std::endl
-                << "dataset: " << data_path << std::endl
-                << "input dtype: " << input_dtype << std::endl
-                << "output dtype: " << output_dtype << std::endl;
+  SolverOutput::get().TurnOn(true);
```

Comment:
as default is ON, this line looks not needed

Reply:
done